### PR TITLE
Use ChartKick gem for charts

### PR DESCRIPTION
**Pull request by Br3nda**
convert sunniness chart to a pie chart.
style up predictions

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
```
     Capybara::Poltergeist::JavascriptError:
       One or more errors were raised in the Javascript code on the page. If you don't care about these errors, you can ignore them by setting js_errors: false in your Poltergeist configuration (see documentation for details).
     
       SyntaxError: JSON Parse error: Unrecognized token '<'
       SyntaxError: JSON Parse error: Unrecognized token '<'
           at http://localhost:8081/assets/charts-1c341fddc0188b2419b74b3bcead385bba0b4ef47432426479a0a465c67724e4.js:18826 in parse
           at http://localhost:8081/assets/charts-1c341fddc0188b2419b74b3bcead385bba0b4ef47432426479a0a465c67724e4.js:18826 in onload
     # ./spec/features/plantings/planting_a_crop_spec.rb:154:in `block (2 levels) in <top (required)>'
```
like the other one

**Comment by Br3nda** [comment]
The error goes away when i remove `// = require chartkick`
Working out why. Might be capybara doesn't like it. Might be something else.

**Comment by Br3nda** [comment]
One test left. Wil sort soon (my laptop is out of action)

**Comment by Br3nda** [comment]
added a "harvested for" and "planted from" chart
image

**Comment by Br3nda** [comment]
GREEN BUILD!

**Comment by Marlena** [comment]
These are so cute! These are graphs I worked on years ago so brava for changing it to chartkick and getting them into a releasable shape. I haven't been active in this project for quite a while so you might want someone more involved to review the code, but I'm really happy to see this going in. Yay!

File: app/assets/stylesheets/predictions.sass [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.predictions
+  .metric
+    height: 180px
+    border: 1px solid lighten($green, 20%)
+    background: $white
+    margin: 4px
+    strong
+      font-size: 250%
+      font-align: center
+    h3
```
Comment (CloCkWeRX) [inline app/assets/stylesheets/predictions.sass]: That seems odd, missing content?

File: config/initializers/assets.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Be sure to restart your server when you modify this file.
```
Comment (CloCkWeRX) [inline config/initializers/assets.rb]: leftover from rails 5 stuff?

Comment (Br3nda) [reply config/initializers/assets.rb]: yes, it was  --- but if i remove it that removal gets merged into rails-5 branch.

**Comment by Marlena** [comment]
🎉🎉🎉

**Pull request merged**